locales/zh_CN/zh_CN.json

```diff
@@ -251,44 +251,44 @@
     "Hold Up!": "等等！",
-    "Holy See (Vatican City State)": "Vatican City",
+    "Holy See (Vatican City State)": "梵蒂冈城",
     "Honduras": "洪都拉斯",
     "Hong Kong": "香港",
     "Hungary": "匈牙利",
-    "I accept the terms of service": "I accept the terms of service",
+    "I accept the terms of service": "我接受服务条款",
     "I agree to the :terms_of_service and :privacy_policy": "我同意 :terms_of_service 和 :privacy_policy",
     "Iceland": "冰岛",
     "ID": "ID",
     "If necessary, you may log out of all of your other browser sessions across all of your devices. Some of your recent sessions are listed below; however, this list may not be exhaustive. If you feel your account has been compromised, you should also update your password.": "如有必要，您可以注销您其他设备上所有的浏览器会话。下面列出了您最近的一些会话，但是，这个列表可能并不详尽。如果您认为您的账户已被入侵，您还应该更新您的密码。",
     "If you already have an account, you may accept this invitation by clicking the button below:": "如果您已经有一个账户，您可以通过点击下面的按钮接受这个邀请：",
     "If you did not create an account, no further action is required.": "如果您未注册帐号，请忽略此邮件。",
-    "If you did not expect to receive an invitation to this team, you may discard this email.": "如果你没有想到会收到这个团队的邀请，你可以丢弃这封邮件。",
-    "If you did not receive the email": "If you did not receive the email",
+    "If you did not expect to receive an invitation to this team, you may discard this email.": "如果您没有想到会收到这个团队的邀请，您可以丢弃这封邮件。",
+    "If you did not receive the email": "如果您没有收到电子邮件",
     "If you did not request a password reset, no further action is required.": "如果您未申请重设密码，请忽略此邮件。",
     "If you do not have an account, you may create one by clicking the button below. After creating an account, you may click the invitation acceptance button in this email to accept the team invitation:": "如果您还没有账号，可以点击下面的按钮创建一个账号。创建账户后，您可以点击此邮件中的邀请接受按钮，接受团队邀请：",
-    "If you need to add specific contact or tax information to your receipts, like your full business name, VAT identification number, or address of record, you may add it here.": "If you need to add specific contact or tax information to your receipts, like your full business name, VAT identification number, or address of record, you may add it here.",
-    "If you're having trouble clicking the \":actionText\" button, copy and paste the URL below\ninto your web browser:": "If you're having trouble clicking the \":actionText\" button, copy and paste the URL below\ninto your web browser:",
+    "If you need to add specific contact or tax information to your receipts, like your full business name, VAT identification number, or address of record, you may add it here.": "如果您需要在收据中添加特定的联系信息或税务信息，例如您的完整公司名称、增值税识别号或记录地址，您可以在此处添加。",
+    "If you're having trouble clicking the \":actionText\" button, copy and paste the URL below\ninto your web browser:": "如果您单击 “:actionText” 按钮时遇到问题，请复制下方链接到浏览器中访问",
     "If you’re having trouble clicking the \":actionText\" button, copy and paste the URL below\ninto your web browser:": "如果您点击「:actionText」按钮时出现问题，请复制下方链接到浏览器中访问：",
```

Comment:
```suggestion
    "If you’re having trouble clicking the \":actionText\" button, copy and paste the URL below\ninto your web browser:": "如果您单击 “:actionText” 按钮时遇到问题，请复制下方链接到浏览器中访问：",
```

The difference between the keys on lines 279 and 280 is that different apostrophe characters are used:
279: `If you're`
280: `If you’re`

Reply:
I thought the original translation here was fine, so I restored it.

Reply:
@andrey-helldar https://github.com/Laravel-Lang/lang/blob/master/source/en.json#L10